src/ngraph_rewrite_pass.cc

```diff
@@ -128,6 +128,13 @@ mutex NGraphRewritePass::s_serial_counter_mutex;
 class NGraphVariableCapturePass : public NGraphRewritePass {
  public:
   Status Run(const GraphOptimizationPassOptions& options) override {
+    // If NGRAPH_TF_DISABLE is set we will not do anything; all subsequent
+    // passes become a no-op.
+    //
+    if (std::getenv("NGRAPH_TF_DISABLE") != nullptr) {
+      return Status::OK();
+    }
+
```

Comment:
Please move this check after the Precaptured graph is dumped. That way, we can look at the state of the graph through tf execution also. This would be useful for debugging purpose.

We can then run TF with NGRAPH disabled and see the Graphs after TF
- Placement as (precaptured)
- RFE as (unmarked)

Reply:
You mean with NGRAPH disabled, correct?

Reply:
yes, corrected.